src/reference/config/catalogs/catalog-karapace.md

```diff
@@ -1,21 +1,21 @@
 ---
-shortTitle: schema-registry
-description: Zilla runtime schema-registry catalog
+shortTitle: karapace
+description: Zilla runtime karapace catalog
 category:
   - Catalog
 ---
 
-# schema-registry Catalog
+# karapace Catalog
 
 ::: important Feature is in Incubator
 Read how to [enable incubator features](../../../how-tos/deploy-operate.md#enable-incubator-features). Star and watch the [Zilla repo](https://github.com/aklivity/zilla/releases) for new releases!
 :::
```

Comment:
Please review all features marked incubator as many features moved out of incubator during the last release.